.github/workflows/publish-techdocs.md

```diff
@@ -32,11 +32,11 @@ jobs:
 
 ## Inputs
 
-| Name                        | Type    | Description                                                                   |
-| --------------------------- | ------- | ----------------------------------------------------------------------------- |
-| `namespace`                 | string  | The entity's namespace within EngHub (usually `default`)                      |
-| `kind`                      | string  | The kind of the entity in EngHub (usually `component`)                        |
-| `name`                      | string  | The name of the entity in EngHub (usually matches the name of the repository) |
-| `default-working-directory` | string  | The directory where the techdocs-cli should be run (default: `.`)             |
-| `checkout-repo`             | boolean | Should this workflow also check out the current repository? (default: `true`) |
-| `publish`                   | boolean | Enable disable publishing the built docs to EngHub (default: `true`)          |
+| Name                        | Type    | Description                                                                             |
+| --------------------------- | ------- | --------------------------------------------------------------------------------------- |
+| `namespace`                 | string  | The entity's namespace within EngHub (usually `default`)                                |
+| `kind`                      | string  | The kind of the entity in EngHub (usually `component`)                                  |
+| `name`                      | string  | The name of the entity in EngHub (usually matches the name of the repository)           |
+| `default-working-directory` | string  | The directory where the techdocs-cli should be run (default: `.`)                       |
+| `publish`                   | boolean | Enable disable publishing the built docs to EngHub (default: `true`)                    |
```

Comment:
```suggestion
| `publish`                   | boolean | Whether to publish the built docs to EngHub (default: `true`)                    |
```

(sorry that's probably going to mess up the alignment of the table)

Reply:
Thank goodness for extensions 🙏